modules/ding_debt_easy_reports/css/ding_debt_easy_reports.admin.css

```diff
@@ -0,0 +1,20 @@
+.ding-debt-easy-report .status-failed {
+  background-color: #f08080;
+}
+
+.ding-debt-easy-report .status-pending {
+  background-color: #ffdab9;
+}
+
+.ding-debt-easy-report .status-reserved {
+  background-color: #b0e0e6;
+}
+
+.test-mail-send-btn {
+  margin-top: 1em;
+}
+
+.mail-cron .form-item {
+  float: left;
```

Comment:
Is the time right to drop floats and clearfixes and move to CSS grid instead?

Reply:
naaa, not right now for the admin css for this one page.